dvc/remote/base.py

```diff
@@ -33,14 +33,16 @@ def __init__(self, msg):
 
 class RemoteBase(object):
     REGEX = None
+    REQUIRES = []
 
     def __init__(self, project, config):
         pass
 
     @classmethod
-    def supported(cls, config):
+    def supported(cls, config, check_dependencies=False):
```

Comment:
I don't think `check_dependencies` flag is required, we can safely always assume that it is True.